pkg/controller/migmigration/backup.go

```diff
@@ -490,6 +490,52 @@ func (t *Task) areStagePodsCreated(resticAnnotationCount int) (bool, error) {
 	return false, nil
 }
 
+// addLabelsToNamespace forces velero to include namespaces
+// in the second backup to avoid creating an empty backup
+func (t *Task) addLabelsToNamespace() error {
+	client, err := t.getSourceClient()
+	if err != nil {
+		log.Trace(err)
+		return err
+	}
+	uniqueBackupLabelKey := fmt.Sprintf("%s-%s", pvBackupLabelKey, t.Owner.UID)
+	namespace := t.PlanResources.MigPlan.Spec.Namespaces[0]
```

Comment:
Do we need to check len(Namespaces) here before assuming Namespaces[0] exists?

Reply:
I just checked and it is possible to have a migplan with an empty Namespace list, although that results in a critical condition rendering the migplan not ready. Still, it's probably safer here to ensure that there is at least one namespace before attempting to add a label to it.